src/config.py

```
@@ -10,6 +10,18 @@
 # PandaScore API Key
 PANDASCORE_API_KEY = os.getenv("PANDASCORE_API_KEY")
 
+
+# Default games to sync/poll when no per-guild configuration is present.
+# Comma-separated env var supported (e.g. "lol,cs2").
+def _parse_default_games(env_val: str | None):
```

Ruff (0.15.5)

[warning] Missing return type annotation for private function `_parse_default_games`

(ANN202)